README.md

```diff
@@ -516,6 +524,36 @@ Leave a blank line between a block and the next statement.
   return obj;
 ```
 
+Each line of code should be no more than 100 characters. This helps with readability in an IDE as well as in pull requests. This includes all lines including strings, template literals, and comments.
+
+```javascript
+  // bad
+  const amazingPeople = { mathematicians = ['Katherine Johnson', 'Dorothy Vaughan'], scientists: ['Alice Ball', 'Mary Elliott Hill'], astronauts: ['Mae C. Jemison', 'Stephanie Wilson']};
+
+  // good
+  const amazingPeople = {
+    mathematicians = ['Katherine Johnson', 'Dorothy Vaughan'],
+    scientists: ['Alice Ball', 'Mary Elliott Hill'],
+    astronauts: ['Mae C. Jemison', 'Stephanie Wilson']
+  };
+
+  // bad
+  // This is a really long comment that should really be written in multiple lines but instead has been written in a single line.
+
+  // good
+  // This is a really long comment that has been written on multiple lines instad of on a single
+  // line so that it is easier to read.
+
+  // bad
+  const str = `this is a really long string that should really written over multiple lines but instead has been written in a single line`;
+
+  // good
+  const str = `this is a really long string that has been written over multiple lines instead
```

Comment:
I disagree here. This assumes a minimum editor width. Anything less than the line length makes it harder to read. Example:

Github diff view: 
![Screen Shot 2020-09-01 at 1 17 49 PM](https://user-images.githubusercontent.com/384629/91901851-a3a38780-ec55-11ea-9286-2b2f0e976e7f.png)

My current editor view: 
![Screen Shot 2020-09-01 at 1 17 25 PM](https://user-images.githubusercontent.com/384629/91901903-b7e78480-ec55-11ea-98a1-aaa331ae6525.png)

The "good" example is more difficult to read. I would exclude scenarios where this type of issue cannot be controlled.

Reply:
This is a good point but I'm not sure where the balance point is for this.  SonarQube will complain if any line is more than 180 columns.  ESlint defaults this value to 80 and AirBnB recommends 100.  I think 80 would likely solve the issue you're showing in your screen captures.  What do you think?

Reply:
I updated the max Len to 180 based on our conversation.